diff -ur orig-ruby-mysql-0.2.5/lib/mysql.rb ruby-mysql-0.2.5/lib/mysql.rb
--- orig-ruby-mysql-0.2.5/lib/mysql.rb	2004-10-06 09:34:27.000000000 -0600
+++ ruby-mysql-0.2.5/lib/mysql.rb	2004-11-05 15:41:15.115354312 -0700
@@ -17,7 +17,7 @@
   require "#{dir}/mysql/protocol"
 
   VERSION            = 20900               # Version number of this library
-  MYSQL_UNIX_PORT    = "/tmp/mysql.sock"   # UNIX domain socket filename
+  MYSQL_UNIX_PORT    = "/var/lib/mysql/mysql.sock"   # UNIX domain socket filename
   MYSQL_TCP_PORT     = 3306                # TCP socket port number
 
   attr_reader :charset               # character set of MySQL connection
